engine/src/hopeit/server/web.py

```diff
@@ -173,6 +174,9 @@ async def stream_startup_hook(app_config: AppConfig, *args, **kwargs):
     """
     app_engine = runtime.server.app_engines[app_config.app_key()]
     for event_name, event_info in app_engine.effective_events.items():
+        if app_engine.app_config.groups and \
```

Comment:
This logic can be simplified for a single group, use a normal if + and statements instead of skipping the unneded ones.